README.md

```diff
@@ -1,5 +1,109 @@
-# Codess in the classroom attendee material
+# Codess In The Classroom
 
-Repository to store the specs and instructions to use in the classroom.
+Welcome! This repository contains the Codess In The Classroom course
+materials. From here you will be able to find (and contribute to!) the
+materials we use in our hands-on workshops.
 
-1. [Week 1](./week1-webclient.md)
+## Introduction
+
+The course is going to be structured as a series of workshops, during which
+you will learn about various technologies and practices used in modern
+software development.
+
+In each workshop, you and your peers will be the development team working on
+part of a web application for a pet adoption center. You will learn about
+front-end web development, APIs, testing and continuous integration, as well
+as how to deploy web applications to the cloud, and how to collaborate in a
+modern software team.
+
+During the workshop, you will have access to a group of mentors, who will be
```

Comment:
nit: I'd suggest removing the second comma here, generally when commas are used in pairs they indicate that the text between them may be skipped and the sentence should continue to read naturally.

Reply:
👍